apps/web/lib/hooks/useInViewObserver.ts

```diff
@@ -26,7 +23,7 @@ export const useInViewObserver = (
           }
         },
         {
-          root,
+          root: root ?? document.body,
```

Comment:
@Udit-takkar It is the fix for failing build as document was being used during SSR earlier